### PR TITLE
socket writer and test

### PR DESCRIPTION
**Pull request by whaley-ethan**

### CODE REVIEW

File: test/echo_server/core_test.clj [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
   (:require [clojure.test :refer :all]
-            [echo_server.core :refer :all]))
+            [echo_server.core :refer :all])
+  (:import (java.io ByteArrayOutputStream BufferedWriter OutputStreamWriter)))
 
 (deftest a-test
   (testing "one equals one"
     (is (= 1 1))))
+
+(deftest writer-test
```
Comment (ericmeyer) [inline test/echo_server/core_test.clj:6]: What happens you write multiple messages sequentially?

File: test/echo_server/core_test.clj [hunk 2]
```diff
@@ -1,7 +1,18 @@
 (ns echo_server.core-test
   (:require [clojure.test :refer :all]
-            [echo_server.core :refer :all]))
+            [echo_server.core :refer :all])
+  (:import (java.io ByteArrayOutputStream BufferedWriter OutputStreamWriter)))
 
 (deftest a-test
```
Comment (ericmeyer) [inline test/echo_server/core_test.clj]: Can this test be deleted?

Comment (whaley-ethan) [reply test/echo_server/core_test.clj]: done

File: test/echo_server/core_test.clj (hunk 1 above)
Comment (whaley-ethan) [reply test/echo_server/core_test.clj:6]: additional test added to codify expected behavior

**Pull request merged**